build-system/app-index/template.js

```diff
@@ -24,8 +24,33 @@ const documentModes = require('./document-modes');
 const html = require('./html');
 const ProxyForm = require('./proxy-form');
 const {KeyValueOptions} = require('./form');
+const {many} = require('./html-helpers');
 const {SettingsModal, SettingsOpenButton} = require('./settings');
 
+const fileListEndpointPrefix = '/dashboard/api/listing';
+
+const examplesPathRegex = /^\/examples\//;
```

Comment:
Can we move these regex into its own file, or constants object? 🤔

Reply:
It's a little cluttered at the moment but I plan to split this file semantically. That way constants will retain meaning without being exported or cluttering.